adaptor/elasticsearch/elasticsearch.go

```diff
@@ -109,12 +111,14 @@ func setupWriter(conf *Elasticsearch) (client.Writer, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	v, err := version.NewVersion(stringVersion)
 	if err != nil {
 		return nil, client.VersionError{URI: conf.URI, V: stringVersion, Err: err.Error()}
 	}
-
+	var parentIdentifier = "elastic_parent"
+	if conf.ParentID != "" {
+		parentIdentifier = conf.ParentID
+	}
```

Comment:
this block is not necessary since the writer doesn't do a check for whether ParentID is an empty string, we can simplify by removing this bit of code and just call `ParentID: conf.ParentID` below.